lib/heuristics/concerns/scheduling_data_initialisation.rb

```diff
@@ -311,4 +309,22 @@ def compute_capacities(quantities, vehicle, available_units = [])
 
     capacities
   end
+
+  def exist_common_multiplexer?(lapses)
+    # If lapses have common multiplexer then any of these
+    # services will be assigned at the same time as service
+    # with smallest lapse (and normaly biggest number of visits).
+    # If this is not the case, we can not guarantee
+    # same_point_day constraint.
+    lapses = lapses.compact.uniq
```

Comment:
This looks like a side-effect, are we sure we want to override the original list? If we call the function with a proper variable in the future, it will modify the variable.